Makefile

```diff
@@ -1,23 +1,117 @@
-install-onnx-dev:
+PYLINT_VERSION := 2.13.9
```

Comment:
a widely used practice in Python repos is `make style` that fixes code structure and imports automatically. It is based on black and isort. Can we have the same?
Here is an example:
```
style_check:
	black --check .
	isort --check .

style:
	black .
	isort .
```

Reply:
@AlexKoff88 that it is used in transformers doesn't automatically mean that this is a widely used practice. 

It would, however, be nice to have a similar functionality. I considered adding `black` before, but the resulting style is pretty much unconfigurable and goes against our current code style at least in terms of used string quoting symbols. Furthermore, `black` and `isort` frequently [conflict with each other](https://github.com/PyCQA/isort/issues/1518), and also running black/isort does not guarantee that pylint will then pass.

I suggest to keep style out of this PR and review the options for style improvements at a later date.

Reply:
@AlexKoff88, @vshampor  I will try  to cover this aspect in another PR.